client/state/data-layer/wpcom/read/following/mine/delete/test/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External Dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal Dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { follow, unfollow } from 'state/reader/follows/actions';
+import { requestUnfollow, receiveUnfollow, unfollowError } from '../';
+
+describe( 'requestUnfollow', () => {
+	it( 'should dispatch a http request', () => {
+		const dispatch = spy();
+		const next = spy();
+		const action = unfollow( 'http://example.com' );
+		requestUnfollow( { dispatch }, action, next );
+		expect( dispatch ).to.have.been.calledWith( http( {
+			method: 'POST',
+			path: '/read/following/mine/delete',
+			apiVersion: '1.1',
+			body: {
+				url: 'http://example.com',
+				source: 'calypso',
+			},
+			onSuccess: action,
+			onFailure: action,
+		} ) );
+		expect( next ).to.have.been.calledWith( action );
```

Comment:
Is it okay to include this here or should it have its own test. ie.
it(' should dispatch to next' , ()=> ....